source/Octopus.Tentacle/Kubernetes/Scripts/RunningKubernetesJob.cs

```diff
@@ -417,7 +417,7 @@ void RecordScriptHasCompleted(int exitCode)
                 var scriptState = stateStore.Load();
                 scriptState.Complete(exitCode);
                 stateStore.Save(scriptState);
-                WriteInfo(writer, $"Job '{jobName}' has completed with exit code {exitCode}");
+                WriteInfo(writer, $"Kubernetes Job '{jobName}' completed with exit code {exitCode}");
```

Comment:
I forgot to change this in my suggestion
```suggestion
                WriteVerbose(writer, $"Kubernetes Job '{jobName}' completed with exit code {exitCode}");
```